packages/components/src/styles/common-styles.ts

```diff
@@ -5,18 +5,43 @@ import type { ThemedColors } from './';
 import {
   borderWidthBase,
   frostedGlassStyle,
+  motionDurationModerate,
+  motionDurationShort,
+  motionDurationLong,
+  motionDurationVeryLong,
+  motionEasingBase,
+  motionEasingIn,
+  motionEasingOut,
   themeDarkBackgroundShading,
   themeLightBackgroundShading,
   themeLightStateFocus,
 } from '@porsche-design-system/utilities-v2';
 import { getThemedColors, prefersColorSchemeDarkMediaQuery } from './';
 import { isThemeDark } from '../utils';
+import type * as fromMotionType from '@porsche-design-system/utilities-v2/dist/esm/motion';
 
-export const transitionDuration = 'var(--p-transition-duration, .24s)';
-const transitionTimingFunction = 'ease';
+const motionMap: Record<keyof typeof fromMotionType, string> = {
+  motionDurationShort,
+  motionDurationModerate,
+  motionDurationLong,
+  motionDurationVeryLong,
+  motionEasingBase,
+  motionEasingIn,
+  motionEasingOut,
+};
+
+export const cssVariableTransitionDuration = '--p-transition-duration';
+export const cssVariableAnimationDuration = '--p-animation-duration';
 
-export const getTransition = (cssProperty: keyof PropertiesHyphen): string =>
-  `${cssProperty} ${transitionDuration} ${transitionTimingFunction}`;
+export const getTransition = (
+  cssProperty: keyof PropertiesHyphen,
```

Comment:
If this was also typed as `VendorLonghandPropertiesHyphen` we could use it for the `-webkit-...` cases, too instead of relying on other workarounds.